### DISCUSSION TITLE
Question: Does DoD Short wipe an SSD securely? If not, what methods should be used for an SSD?

### DISCUSSION THREAD

**Discussion by Ryan-A-Bradley** [PartialVolume/shredos.x86_64#82] [Q&A]
Does DoD Short wipe an SSD securely? If not, what methods should be used for an SSD?

**Comment by PartialVolume** [comment]
This is just my own personal opinion and others may agree or disagree but I wouldn't generally do a DOD short wipe on an SSD. I believe it will securely wipe it, especially as one of the passes is a PRNG stream with all pass verification. I just would avoid using it because of the possibility it could reduce the life span of the disc. Again there's a lot of discussion about this subject. I would love somebody that's got a load of money to buy many different makes and models of SSD and wipe them until destruction to see how many full disc wipes it takes to destroy a disk then publish the data for all to see.

There is a huge amount of discussion on this subject if you look through the open/closed issues on the nwipe GitHub pages but for what it's worth here's what I would do.

Do a ATA secure erase using hdparm first (nwipe currently doesn't do a ATA secure erase but will be able to do one in an upcoming release.) Then I would use hexedit to quickly check whether zeros or ones are present on the wiped disc. Then I would use ShredOS/nwipe to verify the drive using the Verify ones or verify zeros method.

Alternatively some people say you should do a 'write ones' method followed by a verify ones. [corrected] using ShredOS.

If you are paranoid about wiping the disc maybe a secure erase, then a PRNG stream with verification then a ones with verification. I would try to keep such wipes as infrequently as possible to avoid shortening the life of the drive too much. I would probably only use this on disposal or transfer of the drive.

But generally it would be secure erase then ones or zeros  verification using ShredOS.

**Comment by pogue** [comment]
I still like this idea as an alternative. I've seen it recommended SO MANY times to encrypt an SSD with a random key then format it.

[Secure Wiping of SSD Drives using Encryption & Formatting](https://bitbin.it/aJOhwukR/)

Sorry I had to post this on bitbin, I keep getting the error from Github "You can't comment at this time."

**Reply by Firminator** [reply]
Please provide a direct link without without url shorteners. No one knows what's behind them.

**Reply by pogue** [reply]
That's not a URL shortener. It's a pastebin type site called BitBin.

[https://ibb.co/pKwR6dz](https://ibb.co/pKwR6dz)

It's running on a modified version of [Pastebin Django](https://github.com/Matoking/pastebin-django/)

**Reply by Firminator** [reply]
Expecting a link to a research paper, or some other vetted source. bitbin.it or ibb.co seems to be unwanted websites.

**Comment by pogue** [comment]
> Does DoD Short wipe an SSD securely? If not, what methods should be used for an SSD?

No, it does not. So far the best recommended method are best followed in these instructions [Secure Wiping of SSD Drives using Encryption & Formatting](https://bitbin.it/aJOhwukR/)

**Comment by malvarenga123** [comment]
Can nwipe already do an ATA secure erase?

**Reply by PartialVolume** [reply]
No, not yet, that is planned for the next release as an additional method.

**Reply by MallocArray** [reply]
> No, not yet, that is planned for the next release as an additional method.

Still planned for a future release? I saw one from June that I didn't see mentioned in the release notes for that version
